### PR TITLE
Add fly.io deploy docs

### PR DESCRIPTION
**Pull request by pragmaticanon**
This PR adds minimal docs to deploy serverless postgres on fly.io. Here from @kiwicopple's tweet, willing to contribute more and would love to know what's the next milestone in the roadmap for is :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,13 +64,57 @@ After this, you the Oriole background workers will store the data in Tigris. You
...
+
+Deploy your app to Fly:
+
+```bash
+fly deploy --ha=false
+```
+
+This command will use the `fly.toml` file in the project directory to configure and deploy the app.
+
+### Step 4: Allocate a shared IPv4 address
```
Comment (kiwicopple) [inline README.md]:
```suggestion
### Step 4: Allocate an IP address
```

File: README.md [hunk 2]
```diff
@@ -64,13 +64,57 @@ After this, you the Oriole background workers will store the data in Tigris. You
...
+
+To make it easier to connect to your Postgres instance, allocate a shared IPv4 address:
```
Comment (kiwicopple) [inline README.md]:
```suggestion
To make it easier to connect to your Postgres instance, allocate an IP address:
```

**Comment by kiwicopple** [comment]
Awesome PR @pragmaticanon - if you are interested to continue working on this perhaps you can set up github sponsors, and I can sponsor you to continue working on this repo

> would love to know what's the next milestone in the roadmap for is

I can create some issues (but also you can create some if you have ideas and we can discuss them on github)

**Comment by dhanushreddy291** [comment]
@kiwicopple bring up the issues, anyone on the internet can pick it up as well :)

**Pull request merged**